tools/tcpaccept.py

```diff
@@ -258,24 +295,50 @@ def print_ipv6_event(cpu, data, size):
         inet_ntop(AF_INET6, event.saddr).encode(),
         event.lport))
 
+def depict_cnt(counts_tab):
+    for k, v in sorted(
+        counts_tab.items(), key=lambda counts: counts[1].count, reverse=True
+    ):
+        printb(b"%-7d %-12.12s  %-10d" % (k.pid,v.name, v.count))
+    if counts_tab.items():
+        print("\n")
+
 # initialize BPF
 b = BPF(text=bpf_text)
 
 # header
-if args.time:
-    print("%-9s" % ("TIME"), end="")
-if args.timestamp:
-    print("%-9s" % ("TIME(s)"), end="")
-print("%-7s %-12s %-2s %-16s %-5s %-16s %-5s" % ("PID", "COMM", "IP", "RADDR",
-    "RPORT", "LADDR", "LPORT"))
+if args.count:
+    print("Tracing accept count... Hit Ctrl-C to end")
```

Comment:
This printing is conditional, but on BPF side, the data is captured unconditionally.
That is, we pay extra overhead if we don't specify `-c`.

Reply:
It's a good point. This may introduce some overhead in eBPF code. Since the overhead is only at tcp accept with eBFP monitor, I think it should be acceptable?